plugin/src/withSafeKeyboardDetector.ts

```diff
@@ -0,0 +1,14 @@
+import {
+    AndroidConfig
+} from "@expo/config-plugins";
+import { ExpoConfig } from "@expo/config-types";
+  
+
+
+export const withSafeKeyboardDetector = (config: ExpoConfig) => {
```

Comment:
I chose to use default exports for the other plugins because this allows them to be used independendtly if needed, let's do the same here